src/routes/ProtectedRoute.tsx

```diff
@@ -0,0 +1,13 @@
+import { IProtectedProps } from '@/shared/types/routes';
+import { userStore } from '@/store/store';
+import { Navigate } from 'react-router';
+
+const Protected = ({ children }: IProtectedProps) => {
```

Comment:
@BardhMustafa can we keep consistency of naming, since you did PrivateRoute can we make this component name ProtectedRoute